app/models/service.rb

```diff
@@ -217,6 +217,7 @@ def self.all
   def using_proxy_pro?
     provider_can_use?(:proxy_pro) && proxy.self_managed?
   end
+  delegate :service_preffix, to: :provider, allow_nil: true
```

Comment:
I do not understand :confused: 
https://github.com/3scale/porta/blob/e2dbb0cf87ed7a340261f681b9444254dd1a199b/app/models/account.rb#L6-L8

Reply:
Right it should be removed 👍